### DISCUSSION TITLE
Fade out / Opacity of the hidden words - zprůhlednění konců vět

### DISCUSSION THREAD

**Discussion by kotyzap** [met/subadub#20] [Ideas]
What about instead of using "-" for every word hidden offer to a user option to set the opacity for the hidden part...

For those who are not that good, it might give a chance to be "saved by the... almost invisible but still readable" text.

Czech: Co nabídnout možnost neskrývat konec vět tím rušivým znaménkem "mínus", ale zobrazit ta slova téměř transparentně, takže pokud nezachytím audio, tak se mohu zachránit přečtením.

**Comment by met** [comment]
Not sure yet, but I can try and experiment with this. Thanks 👍
